src/src/pages/capabilities/AwsResourceCount/index.js

```diff
@@ -11,24 +11,27 @@ import {
   TableHead,
   TableHeaderCell,
   TableRow,
+  Text,
 } from "@dfds-ui/react-components";
+import { MaterialReactTable } from "material-react-table";
 
 export function InlineAwsCountSummary({ data }) {
-    return (
-      <>
-        <div>
-          <img src={awsLogo} alt="AWS icon" style={{ height: "1rem" }} />
-          <span> {data}</span>
-        </div>
-      </>
-    );
-  }
+  return (
+    <>
+      <div>
+        <img src={awsLogo} alt="AWS icon" style={{ height: "1rem" }} />
+        <span> {data}</span>
+      </div>
+    </>
+  );
+}
 
 export function DetailedAwsCountSummary({ capabilityId }) {
   const { metricsWrapper } = useContext(AppContext);
   const [showResourcesWindow, setShowResourcesWindow] = useState(false);
-  const count = metricsWrapper.getAwsResourcesTotalCountForCapability(capabilityId);
-  
+  const count =
+    metricsWrapper.getAwsResourcesTotalCountForCapability(capabilityId);
```

Comment:
I think our line length linting rule is set to too few characters, if this is the result.

Reply:
This would be 85 characters if on a single line, if I am not mistaken.
80 was a good standard back in the day, but I believe it is slightly too restrictive